code/src/components/Game.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux'
+
+import { rooms } from '../reducers/rooms'
+import { loader } from '../reducers/loader'
+
+import arrow_one from '../assets/arrow-one.svg'
+import arrow_split from '../assets/arrow-split.svg'
```

Comment:
Just a tiny note, here there are some semi-colons missing. Noticed that you used them above and below 😊